projects/grafana/setup-grafana.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+datasource_name=$1
+prometheus_namespace=$2
+
+usage() {
+echo "
+USAGE
+    setup-grafana.sh pro-ocp openshift-metrics true
+
+    args:
+      datasource_name: grafana datasource name
+      prometheus_namespace: existing prometheus name e.g openshift-metrics
+
+    note:
+      the project must have view permissions for kube-system
+"
+exit 1
+}
+
+[[ -n ${datasource_name} ]] || usage
+
+oc new-project grafana
+oc process -f grafana-ocp.yaml |oc create -f -
+oc rollout status deployment/grafana-ocp
+oc adm policy add-role-to-user view -z grafana-ocp -n kube-system
+
+payload="$( mktemp )"
+cat <<EOF >"${payload}"
+{
+	"name": "${datasource_name}",
+	"type": "prometheus",
+	"typeLogoUrl": "",
+	"access": "proxy",
+	"url": "https://$( oc get route prometheus -n "${prometheus_namespace}" -o jsonpath='{.spec.host}' )",
+	"basicAuth": false,
+	"withCredentials": false,
+	"jsonData": {
+		"tlsSkipVerify":true,
+		"token":"$( oc sa get-token grafana-ocp )"
+	}
+}
+EOF
+
+grafana_host="https://$( oc get route grafana-ocp -o jsonpath='{.spec.host}' )"
+curl -H "Content-Type: application/json" -u admin:admin "${grafana_host}/api/datasources" -X POST -d "@${payload}"
+
+dashboard_file="./openshift-cluster-monitoring.json"
```

Comment:
As per the discussion in https://github.com/openshift/release/pull/425 let's move forward with your PRs in origin and reuse the dashboard from origin here.

Reply:
ok, until the origin PR will merge, let use it here ?!
I'll add a TODO to remove it.